user_journey/user_journey.model.lkml

```diff
@@ -75,18 +75,34 @@ explore: funnel_analysis {
 }
 
 explore: cohort_analysis {
-  view_label: " User-Days"
+  view_label: " User Properties"
   from: cohort
-  join: message_ids {
-    relationship: many_to_one
+  join: cohort_event_type_1 {
+    view_label: "Cohort 1"
+    relationship: one_to_one
+    type: cross
+  }
+  join: cohort_event_type_2 {
+    view_label: "Cohort 2"
+    relationship: one_to_one
+    type: cross
+  }
+  join: cohort_event_type_3 {
+    view_label: "Cohort 3"
+    relationship: one_to_one
+    type: cross
+  }
+  join: cohort_event_type_4 {
+    view_label: "Cohort 4"
+    relationship: one_to_one
     type: cross
   }
   join: days_since_message {
     relationship: many_to_one
     type: cross
   }
   join: clients_last_seen {
-    view_label: "Days of Use"
+    fields: []
```

Comment:
We now don't expose any fields from clients_last_seen directly. Instead we surface them through each of the cohorts.